src/fetch/FetchExample.re

```diff
@@ -0,0 +1,87 @@
+type dog = string;
+
+type arrayDogs = array(dog);
+
+type state = { dogs: arrayDogs, isLoading: bool, error: bool };
+
+type action =
+  | DogsFetch
+  | DogsFetched(arrayDogs)
+  | DogsFailedToFetch;
+
+module Decode = {
+  let dogs = json : arrayDogs =>
+  Json.Decode.(
```

Comment:
This should be indented one mroe level. Alternatively, run refmt on it to make it look consistent, and completely different.